metadata-ingestion/docs/sources/dbt/dbt.md

```diff
@@ -38,6 +38,12 @@ meta_mapping:
     operation: "add_terms"
     config:
       separator: ","
+  documentation_link:
+    match: ".*"
+    operation: "add_link"
+    config:
+      link: "https://en.wikipedia.org/wiki/Anchor_text"
```

Comment:
let's change this example to use a pattern

in reality, we don't expect people to just add the same link to every asset. instead, the link is going to be in their meta section and we have to extract it out